cross/postgresql/Makefile

```diff
@@ -0,0 +1,19 @@
+PKG_NAME = postgresql
+PKG_VERS = 10.11
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://ftp.postgresql.org/pub/source/v10.11/
```

Comment:
to simplify future updates use `v$(PKG_VERS)` instead of `v10.11` here

Reply:
indeed fixed